monai/transforms/post/dictionary.py

```diff
@@ -419,6 +419,10 @@ class Invertd(MapTransform):
     A typical usage is to invert the pre-transforms (appplied on input `image`) on the model `pred` data.
 
     Note:
+        According to the `collate_fn`, this transform may return a list of Tensor without batch dim,
+        some following post transforms may not support a list of Tensor, users can leverage the `post_func`
+        arg for basic processing logic.
+
```

Comment:
thus some following .... and users can leverage ....

Reply:
updated in the latest commit.
Thanks.